llama-index-python-vectorstore/patches/pin-test-output.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/tests/test_vectorstore.py b/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/tests/test_vectorstore.py
+index b48869690..a803fb668 100644
+--- a/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/tests/test_vectorstore.py
++++ b/llama-index-integrations/vector_stores/llama-index-vector-stores-mongodb/tests/test_vectorstore.py
+@@ -20,7 +20,7 @@ from .conftest import lock
+ 
+ def test_documents(documents: List[Document]) -> None:
+     """Sanity check essay was found and documents loaded."""
+-    assert len(documents) == 23
++    assert len(documents) == 25
```

Comment:
@blink1073 Do we really want to use a patch for this? Was this change expected?

Reply:
Yes, I commented on the ticket.  This patch will fail once there is a new release of llama-index-core, and we can remove it.